src/config/dynamic-theme-fixes.config

```diff
@@ -4029,6 +4029,15 @@ CSS
 
 ================================
 
+snack.expo.io
+
+CSS
+._1rs24yq {
```

Comment:
`#root > div > div > div:nth-child(2) > div:nth-child(2) > div:nth-child(2) > div > div:nth-child(2)` Their you go